docs/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -243,7 +243,7 @@ service (Apache, Rails, etc.), you would run something like
 recommended for any service-based image.
 
 In most other cases, `CMD` should be given an interactive shell (bash, python,
-perl, etc), for example, `CMD ["perl", "-de0"]`, `CMD ["python"]`, or
+perl, etc.), for example, `CMD ["perl", "-de0"]`, `CMD ["python"]`, or
```

Comment:
Why not say something like:

In most other cases, `CMD` should be an interactive shell, such as Bash, Python, or Perl. The following examples use Perl and Python:
    ```
    CMD ["perl", "-de0"]
    ```
    ```
    CMD ["python"]
    ```